### PR TITLE
fix empty Country text format

### LINKED ISSUE #107: Correctly format Country text when no country given
Notice the random floating `()` in the `Country` field:

```
$ ipinfo 1234::
Core
- IP           1234::
- Anycast      false
- Hostname     
- City         
- Region       
- Country       ()
- Location     
- Organization 
- Postal       
- Timezone     

Privacy
- VPN     false
- Proxy   false
- Tor     false
- Relay   false
- Hosting false
- Service 
```

### PR DESCRIPTION
**Pull request by nicksherron**
closes #107

### CODE REVIEW

File: ipinfo/utils_output.go [hunk 1]
```diff
@@ -141,7 +141,11 @@ func outputFriendlyCore(d *ipinfo.Core) {
 	printline("Hostname", d.Hostname)
 	printline("City", d.City)
 	printline("Region", d.Region)
-	printline("Country", fmt.Sprintf("%v (%v)", d.CountryName, d.Country))
+	country := d.Country
+	if country != "" {
+		country = fmt.Sprintf(" (%v)", country)
+	}
+	printline("Country", d.CountryName+country)
```
Comment (UmanShahzad) [inline ipinfo/utils_output.go:148]: A bit simpler:

```go
if d.Country == "" {
	printline("Country", "")
} else {
	printline("Country", fmt.Sprintf("%v (%v)", d.CountryName, d.Country))
}
```

Comment (nicksherron) [reply ipinfo/utils_output.go:148]: I thought I would cover the case where the country name was present but the country was empty. Will they both always be either present or missing? If that's the case I see your point.

Comment (UmanShahzad) [reply ipinfo/utils_output.go:148]: Yeah they are strongly linked; the API gives us the country (ISO 3166-1 alpha-2 code, I think), and the decoded name is derived from that code on the client side. We can assume they're always gonna be together. In the worst case, country may exist but the name may not for contested countries. Even in those cases (like `XK => Kosovo`) we seem to cover it on the client side.

Comment (UmanShahzad) [reply ipinfo/utils_output.go:148]: @nicksherron lemme know if you'll finalize this, if not I'll handle it on the `master` branch.

**Comment by UmanShahzad** [comment]
Solved in #116

**Pull request closed without merging**